### ISSUE TITLE
View uploaded file

### ISSUE THREAD

**Issue by anik786** [FineUploader/fine-uploader#1696]
## Type of issue [REQUIRED]
feature request / how to do

## Uploader type [REQUIRED]
traditional

### Detailed description of the feature
Currently it is not possible to view the file that was uploaded. Only the file name. The closest we have to this feature is a thumbnail preview for images. However this is not even good enough as we cannot preview the image at full-size. This is extremely problematic if we return to the application at a later date and want to know what files was uploaded. 

An example use case is an app that requires the user to upload a document/s in a PDF format. A user may upload a document as usual. Several days later, the user goes back to this app. The app uses the "addInitialFiles()" method to show the uploaded document/s. The user wants to check if the document he/she uploaded was in fact the correct file. This is currently not possible.

Therefore a solution to this is to have a link that navigates the user to the appropriate file in a new window. This may be a hyperlink on the file name itself, or a button with the text "Open" (similar to the delete button).

Is this type of behaviour possible now by any chance?

**Comment by rnicholus** [comment]
>Is this type of behaviour possible now by any chance?

Yes, see http://stackoverflow.com/questions/40377213/fine-uploader-idiomatic-way-to-add-download-button-to-each-file-uploaded for inspiration.

**Comment by anik786** [comment]
In which case, can we please add this feature to the Core UI

**Comment by rnicholus** [comment]
I'm not sure how this would be implemented in a generic way without adding a notable amount of complexity to the already complex codebase. I'm not generally in favor of adding more code to the library when the goal can be achieve by following an example that utilizes the existing API.

**Comment by anik786** [comment]
I'm a bit confused how to do it using the existing API anyhow. I have looked at your answer in SO, and unsure as to how do this.

I added a "Open" button in the `fine-uploader-row-template` file. 

I also know how to hook on the `onStatusChange` event to check when the status becomes `UPLOAD_SUCCESS`. 

However I cannot seem to find a way to fetch the original response from the server using just the ID, as this is where we would get the URL from.
